src/pages/Transaction.vue

```diff
@@ -328,7 +328,7 @@ export default {
               div(class="col-3")
                 strong {{ `From: ` }}
```

Comment:
this

Reply:
I fixed all the texts that looks like `{{ `From: ` }}`